homework/shelkovich_ilya/circle/src/test/java/com/tutrit/java/quickstart/service/CircleServiceTest.java

```diff
@@ -0,0 +1,26 @@
+package com.tutrit.java.quickstart.service;
+
+import com.tutrit.java.quickstart.bean.Circle;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CircleServiceTest {
+
+    Circle circle;
+    CircleService circleService;
+
+    @Before
+    public void setUp(){
+        this.circle = new Circle(4);
+        this.circleService = new CircleService();
+    }
+
+    @Test
+    public void calculateArea() {
+        double expected = Math.PI * circle.getRadius() * circle.getRadius();
```

Comment:
should be hardcoded value. You not suppose to test your logic using the same logic.

Reply:
> should be hardcoded value. You not suppose to test your logic using the same logic.

thanks! Now I see that in expected same process than in actual. LOL